### PR TITLE
37891 : display the avatar of user and space in news application

### PR DESCRIPTION
**Pull request by NaderSai**
Avatar is now displayed in news application

### CODE REVIEW

File: webapp/src/main/webapp/news/components/NewsApp.vue [hunk 1]
```diff
@@ -65,13 +65,13 @@
...
             <div class="newsOwner">
               <a :href="news.authorProfileURL">
-                <img :src="news.profileAvatarURL">
+                <img :lazy-src="news.authorAvatarUrl" :src="news.authorAvatarUrl" eager/>
                 <span>{{ news.authorFullName }}</span>
               </a>
-              <i v-if="!news.hiddenSpace" class="uiIconArrowNext"></i>
-              <span v-if="!news.hiddenSpace" class="newsSpace">
+              <i class="uiIconArrowNext"></i>
+              <span class="newsSpace">
```
Comment (ahamdi) [inline webapp/src/main/webapp/news/components/NewsApp.vue]: Please revert this change, this should not be removed

File: webapp/src/main/webapp/news/components/NewsApp.vue [hunk 2]
```diff
@@ -65,13 +65,13 @@
           <div class="newsInfo">
             <div class="newsOwner">
               <a :href="news.authorProfileURL">
-                <img :src="news.profileAvatarURL">
+                <img :lazy-src="news.authorAvatarUrl" :src="news.authorAvatarUrl" eager/>
```
Comment (ahamdi) [inline webapp/src/main/webapp/news/components/NewsApp.vue]: lazy-src is not a standard attribute AFAIK

File: webapp/src/main/webapp/news/components/NewsApp.vue [hunk 3]
```diff
@@ -65,13 +65,13 @@
...
                 <span>{{ news.authorFullName }}</span>
               </a>
-              <i v-if="!news.hiddenSpace" class="uiIconArrowNext"></i>
-              <span v-if="!news.hiddenSpace" class="newsSpace">
+              <i class="uiIconArrowNext"></i>
+              <span class="newsSpace">
                 <a :href="news.spaceUrl" class="newsSpaceName">
-                  <img :src="news.spaceAvatarUrl">
+                  <img :lazy-src="news.spaceAvatarUrl" :src="news.spaceAvatarUrl" eager/>
```
Comment (ahamdi) [inline webapp/src/main/webapp/news/components/NewsApp.vue]: lazy-src is not a standard attribute AFAIK

**Review by ahamdi — CHANGES_REQUESTED** [review]

**Review by aelbenney — APPROVED** [review]

**Review by ahamdi — APPROVED** [review]

**Comment by NaderSai** [comment]
Problem: Avatar of the user and the space in which the article is posted are not displayed in News app
How it was solved: read the avatar URL from the class News, not from static URL.
